.evergreen/run-tests.sh

```diff
@@ -43,12 +46,12 @@ provision_ssl () {
   if [ ! -f client.pkc ]; then
     openssl pkcs12 -CAfile ${DRIVERS_TOOLS}/.evergreen/x509gen/ca.pem -export -in ${DRIVERS_TOOLS}/.evergreen/x509gen/client.pem -out client.pkc -password pass:bithere
   fi
-  if [ ! -f mongo-truststore ]; then
-    ${JAVA_HOME}/bin/keytool -importcert -trustcacerts -file ${DRIVERS_TOOLS}/.evergreen/x509gen/ca.pem -keystore mongo-truststore -storepass hithere -storetype JKS -noprompt
-  fi
+
+  cp ${JAVA_HOME}/lib/security/cacerts mongo-truststore
```

Comment:
Instead of creating a branch new trust store start with a copy of the JDK trust store and then add our key to it.  This lets us connect to both MongoDB servers with the self-signed cert as well as AWS Key Management Service which has a proper certificate.